src/components/EditMenu.vue

```diff
@@ -520,14 +528,13 @@
       <div
         v-for="miniWidget in availableMiniWidgetTypes"
         id="mini-widget-card"
-        :ref="(el) => (miniWidgetContainers[miniWidget.component] = el as HTMLElement)"
         :key="miniWidget.hash"
-        class="flex flex-col items-center justify-between w-full rounded-md bg-[#273842] hover:brightness-125 h-[90%] aspect-square cursor-pointer elevation-4 overflow-clip pointer-events-none"
+        class="flex flex-col items-center justify-between rounded-md bg-[#273842] hover:brightness-125 h-[90%] aspect-square cursor-pointer elevation-4 overflow-clip pointer-events-none"
```

Comment:
This is the part that breaks the auto-width.

Reply:
Yes, it is and it's weird how this zombie changes keep being deleted and brought back to life over and over... :smile: